src/sweeper/sweepers/duplicates.py

```diff
@@ -7,52 +7,54 @@
 
 import arcpy
 import re
+import os
 from xxhash import xxh64
 
-
 class DuplicateTest(object):
-    def __init__(self):
+    def __init__(self, workspace, table_name):
+        self.report = {}
+        self.workspace = workspace
+        self.table_name = table_name
 
 
-    def sweep(self, fc):
+    def sweep(self):
 
         digestDict = {}
-        duplicateDict = {}
-        dupePrintLst = []
+
         digTrim = re.compile(r'(\d+\.\d{2})(\d+)')
 
-        shps = ['Shape', 'SHAPE']
+        arcpy.env.workspace = self.workspace
 
-        fields = [f.name for f in arcpy.ListFields(fc) if f.name not in shps]
+        omitFlds = ['Shape', 'SHAPE', 'GLOBALID']
```

Comment:
what do you think about lowercasing all of the fields so it doesn't matter how shape is cased?

Reply:
i like that idea.  i use upper, and i know it has always helped me out.